custom_components/mqtt_vacuum_camera/valetudo/MQTT/connector.py

```diff
@@ -293,6 +303,17 @@
 
             self._shared.rand256_active_zone = rrm_active_segments
 
+    async def async_handle_start_command(self, msg):
+        """fire event vacuum start"""
+        mqtt_command = msg.payload
+        if str(mqtt_command).lower() == "start":
+            # Fire the vacuum.start event when START command is detected
+            self._hass.bus.async_fire(
+                "event_vacuum_start",
+                event_data=mqtt_command,
+                origin=EventOrigin.local,
+            )
+
```

Comment:
@coderabbitai msg.payload is bytes this why I convert it to str. I will remove the mqtt_command variable so that we use only msg.payload in the next commit